WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -691,11 +693,6 @@ title_despot = {
 			has_landed_title = e_byzantium
 			has_landed_title = e_roman_empire
 		}
-		OR = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```

Comment:
@loup99 

-because those conditions are stupid. If you control the Roman Empire, you should be able to use the Roman Minor Titles. _**Period.**_